azurelinuxagent/ga/agent_update_handler.py

```diff
@@ -33,20 +33,32 @@ def get_agent_update_handler(protocol):
     return AgentUpdateHandler(protocol)
 
 
+RSM_UPDATE_STATE_FILE = "rsm_update.json"
```

Comment:
you may want to combine those 2 files

the number of state files keep growing, so there are more items that need to be cleaned up, etc